internal/services/v1/permissions_test.go

```diff
@@ -339,7 +339,7 @@ func TestCheckPermissionWithDebugInfo(t *testing.T) {
 		SchemaString: debugInfo.SchemaUsed,
 	}, &emptyDefaultPrefix)
 	require.NoError(err, "Invalid schema: %s", debugInfo.SchemaUsed)
-	require.Equal(3, len(compiled.OrderedDefinitions))
+	require.Equal(4, len(compiled.OrderedDefinitions))
```

Comment:
👍🏻because the new implementation also takes caveats into consideration

Reply:
Yep